index.bs

```diff
@@ -1462,3 +1484,231 @@ The [=remote end steps=] with |command parameters| are:
     1. Return the result of [=updating the event map=] with [=current session=],
        |list of event names|, |list of contexts| and enabled false.
 </div>
+
+
+## browsingContext ## {#module-browsingContext}
+
+The <dfn export for=modules>Browsing Context</dfn> module contains commands and
+events relating to browsing contexts.
+
+### Definition ### {#module-browsingContext-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+BrowsingContextCommand = (BrowsingContextGetAllCommand)
```

Comment:
```suggestion
BrowsingContextCommand = (BrowsingContextListCommand)
```

Right? And doesn't `BrowsingContextCommand` need to be used somewhere for the CDDL to make sense?